system_reports/MACOS/M1/osx-xcode-13.4.x.log

```diff
@@ -7,29 +7,26 @@
 
 
 === Revision / ID ======================
-* BITRISE_OSX_STACK_REV_ID: v2022_08_11_1
+* BITRISE_OSX_STACK_REV_ID: v2022_08_24_1
 ========================================
 
 
 === Pre-installed tool versions ========
-* Go: go version go1.17.13 darwin/arm64
+* Go: go version go1.18.5 darwin/arm64
 * Python: Python 2.7.18
-* Node.js: v16.16.0
-* NPM: 8.11.0
+* Node.js: v16.17.0
+* NPM: 8.15.0
 * Yarn: 1.22.19
 * Java:   system
   1.8
   1.8.0.292
-  11
```

Comment:
Is it intended to be removed? @bitrise-tom

Reply:
No, it should still be there. We will need to check the machine and see why it's gone.

Reply:
It is a jenv bug that causes it, but we somehow worked around it earlier. The sed included in macOS works different than in linux (which jenv expects)

Reply:
<img width="490" alt="image" src="https://user-images.githubusercontent.com/48553835/187952989-8cd7daaf-b3f8-4b01-aac2-1df452b16ce6.png">

So we definitely need this?